BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift

```diff
@@ -101,6 +101,7 @@ public struct StandardImageIdentifiers {
         public static let subtract = "subtractLarge"
         public static let sync = "syncLarge"
         public static let syncTabs = "syncTabsLarge"
+        public static let tabNumber = "tabNumberLarge"
```

Comment:
Where is this image coming from? The Acorn icon library doesn't have an image named this way. Only images found in Acorn should be added to StandardImageIdentifiers. Please use `tab` instead.

Reply:
I hadn't found it, so I took it from the Client.